specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json

```diff
@@ -123,6 +123,78 @@
         }
       }
     },
+    "/{resourceId}/query": {
+      "get": {
+        "operationId": "Query_ResourceGet",
+        "summary": "Execute an Analytics query using resource URI",
+        "description": "Executes an Analytics query for data in the context of a resource. [Here](https://docs.microsoft.com/azure/azure-monitor/logs/api/azure-resource-queries) is an example for using POST with an Analytics query.",
+        "x-ms-examples": {
+          "simple-query": {
+            "$ref": "examples/oms-get-resource-centric-example.json"
```

Comment:
The "OMS" product name is deprecated, can we please switch to "Log Analytics" ("la-")